recipe/run_test.sh

```diff
@@ -2,6 +2,11 @@
 set -ex
 
 
+if [ $cuda_compiler_version == '11.0' ]; then
+    conda create -n test cudatoolkit=11.2 cutensor
```

Comment:
Could we list these in `test/requires` instead?

Reply:
It's only for debugging for now. Yeah we could move it there once we confirm things are working.

Reply:
I don't think this is a valid test, is it? Does the solver fetch `cutensor` from local channel or conda-forge?

Reply:
The solver treats packages that are built locally as coming from a special channel that has higher priority than any other channel

Reply:
I see, good to confirm.